packages/tx/test/index.ts

```diff
@@ -1,20 +1,19 @@
-import { isTruthy } from '@ethereumjs/util'
 import * as minimist from 'minimist'
 
 const argv = minimist(process.argv.slice(2))
 
-if (isTruthy(argv.b)) {
+if (argv.b === true) {
```

Comment:
I'm good with this but just want to confirm, we're not looking to enforce a hard rule on how we write boolean checks where the value we're checking is indeed a boolean like here, right?  So stylistically for us, it's fine to write either:
`if (argv.b)` or `if (argv.b === true)` right?  Or does this eslint rule disallow the first?

Reply:
You are correct! In this case, since `argv.b`, is not typed, I made it explicit, but in normal cases where we're dealing with a `boolean`, there's no need to be explicit. Explicitness is only needed when truthiness/falsiness is involved (and boolean is true/false, not "truthy/falsy").

Reply:
Ah, that's good to have this confirmed! 🙂